client/lib/site/utils.js

```diff
@@ -122,3 +122,25 @@ export function getUnmappedUrl( site ) {
 
 	return site.options.main_network_site || site.options.unmapped_url;
 }
+
+/**
+ * Returns a filtered array of WordPress.com site IDs where a Jetpack site
+ * exists in the set of sites with the same URL.
+ *
+ * @param {Array} siteList Array of site objects
+ * @returns {number[]} Array of site IDs with URL collisions
+ */
+export function getJetpackSiteCollisions( siteList ) {
```

Comment:
Would it be possible to add some tests around this logic?

Reply:
Sure!, I'll add some tests. Good Idea!
Thanks!

Reply:
@danielbachhuber, I've added the tests here 4977ebe8823138f363c2491257ff1fcc0ef81fc8

Reply:
@sejas Awesome, thanks!